src/Notify/Notifier/AppleNewsNotifier.php

```diff
@@ -156,12 +156,42 @@ protected function updateArticle(Message $notification, Message $app, Message $a
         }
 
         $latestRevision = $this->getLatestRevision($notification);
-        if (null === $latestRevision || $article->get('apple_news_revision') === $latestRevision) {
+        if (null !== $latestRevision && $metadata['revision'] !== $latestRevision) {
+            $metadata['revision'] = $latestRevision;
+            $result = $this->api->updateArticle((string)$article->get('apple_news_id'), $document, $metadata);
+        }
+
+        if ($result['ok']) {
+            return $result;
+        }
+
+        $getArticleResult = $this->api->getArticle((string)$article->get('apple_news_id'));
+        if (!$getArticleResult['ok']) {
+            $this->sendMessageToSlack($article, $notification);
             return $result;
         }
 
-        $metadata['revision'] = $latestRevision;
-        return $this->api->updateArticle((string)$article->get('apple_news_id'), $document, $metadata);
+        $revision = $getArticleResult['response']['data']['revision'];
+        if ($metadata['revision'] !== $revision) {
+            $metadata['revision'] = $revision;
+            $result = $this->api->updateArticle((string)$article->get('apple_news_id'), $document, $metadata);
```

Comment:
@tbone1020 use `fget('apple_news_id')` and remove`(string)`

Reply:
done